### PR TITLE
added Role module and updated app.module

### PR DESCRIPTION
**Pull request by Pai026**

### CODE REVIEW

File: src/role/role.repository.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { EntityRepository, Repository } from 'typeorm';
+import { Role } from './entities/Role.entity';
+
+@EntityRepository(Role)
+export class RoleRepository extends Repository<Role> {
+    async getAllRoles(): Promise<Role[]>{
+        const query = this.createQueryBuilder('role');
+        return await query.getMany();
+    }
+    async getById(id:number): Promise<any> {
```
[the remark this answers is not in the document]
Comment (Pai026) [reply src/role/role.repository.ts:10]: its for finding by userId

File: src/role/role.service.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    private logger = new Logger('Role Service');
+    constructor(
+   @InjectRepository(RoleRepository)
+   private readonly roleRepository: RoleRepository){}
+   
+   async getAllRoles(req: any): Promise<any> {
+    return this.roleRepository.getAllRoles();
+   }
+    async addRoles(data:any): Promise<any> {
+            const role1 = await this.roleRepository.getById(data.userId)
```
[the remark this answers is not in the document]
Comment (Pai026) [reply src/role/role.service.ts:16]: you mean multiple roles?

**Comment by tomahawk-pilot** [comment]
@giju is it mergable ?

**Comment by tomahawk-pilot** [comment]
@Pai026 fix this pr

**Pull request merged**